src/main/java/com/faforever/api/data/domain/BanInfo.java

```diff
@@ -55,8 +56,8 @@ public Player getPlayer() {
   }
 
   @ManyToOne
+  @UpdatePermission(expression = "Prefab.Role.None")
   @JoinColumn(name = "author_id")
-  @NotNull
```

Comment:
Why was @NotNull removed here?

Reply:
It was removed because it is set now anyway so no @NotNull nedded

Reply:
No. Implementations may change, but there is nothing more explicit than validation right on the entity. No reason to remove it. It gives additional information and enforces a validation.